python/ray/rllib/utils/actors.py

```diff
@@ -36,7 +36,11 @@ def completed_prefetch(self):
 
         for worker, obj_id in self.completed():
             plasma_id = ray.pyarrow.plasma.ObjectID(obj_id.id())
-            ray.worker.global_worker.plasma_client.fetch([plasma_id])
+            if not ray.global_state.use_raylet:
+                ray.worker.global_worker.plasma_client.fetch([plasma_id])
+            else:
+                ray.worker.global_worker.local_scheduler_client\
+                    .reconstruct_objects([obj_id], False)
```

Comment:
the second param should be True, which means only fetch objects without requesting  reconstruction.